### PR TITLE
Move Docker Trust out of experimental

### PR DESCRIPTION
**Pull request by n4ss**
Signed-off-by: Nassim 'Nass' Eddequiouaq <eddequiouaq.nassim@gmail.com>

This is a follow-up on https://github.com/docker/cli/pull/934 which moves `docker trust view` to `docker trust inspect --pretty`.

-----------------------------

**- What I did**
Moved `docker trust` commands out of experimental

**- How to verify it**
Run `docker trust` commands successfully without "experimental" enabled in the CLI `config.json`.

**- Description for the changelog**
Moved Docker Trust commands out of experimental.

-----------------------------

**Important: This PR shouldn't be merged before https://github.com/docker/cli/pull/934**

### CODE REVIEW

**Comment by thaJeztah** [comment]
This also requires the code changes to remove it from experimental, for example; https://github.com/docker/cli/blob/2731c71c993a4d1e43101eea56e7c343315024f1/cli/command/trust/cmd.go#L16

**Comment by thaJeztah** [comment]
And possibly documentation changes

**Comment by vieux** [comment]
@thaJeztah I don't understand your comment, `Annotations: map[string]string{"experimentalCLI": ""}, ` was removed.

**Comment by thaJeztah** [comment]
oh, never mind, I thought there were more locations it was in 😊 but we only put it on the root (`trust`) command

**Comment by vieux** [comment]
@thaJeztah I pushed a doc update here

File: docs/reference/commandline/trust_inspect.md [hunk 1]
```diff
@@ -362,3 +357,102 @@ $ docker trust inspect alpine notary
...
+
+### Formatting
+
+You can print the inspect output in a human-readable format instead of the default
+JSON output, by using the `--pretty` option:
+
+### Get details about signatures for a single image tag
+
+```bash
+$ docker trust inspect --pretyy alpine:latest
```
Comment (thaJeztah) [inline docs/reference/commandline/trust_inspect.md]: typo pretyy

File: docs/reference/commandline/trust_inspect.md [hunk 2]
```diff
@@ -362,3 +357,102 @@ $ docker trust inspect alpine notary
...
+Administrative keys for alpine:latest:
+Repository Key:	5a46c9aaa82ff150bb7305a2d17d0c521c2d784246807b2dc611f436a69041fd
+Root Key:	a2489bcac7a79aa67b19b96c4a3bf0c675ffdf00c6d2fabe1a5df1115e80adce
+```
+
+The `SIGNED TAG` is the signed image tag with a unique content-addressable `DIGEST`. `SIGNERS` lists all entities who have signed.
+
+The administrative keys listed specify the root key of trust, as well as the administrative repository key. These keys are responsible for modifying signers, and rotating keys for the signed repository.
+
+If signers are set up for the repository via other `docker trust` commands, `docker trust inspect --pretty` displays them appropriately as a `SIGNER` and specify their `KEYS`:
```
Comment (thaJeztah) [inline docs/reference/commandline/trust_inspect.md]: can you wrap these lines to 80 chars?

File: docs/reference/commandline/trust_inspect.md [hunk 3]
```diff
@@ -362,3 +357,102 @@ $ docker trust inspect alpine notary
...
+latest              1072e499f3f655a032e88542330cf75b02e7bdf673278f701d7ba61629ee3ebe   (Repo Admin)
+
+Administrative keys for alpine:latest:
+Repository Key:	5a46c9aaa82ff150bb7305a2d17d0c521c2d784246807b2dc611f436a69041fd
+Root Key:	a2489bcac7a79aa67b19b96c4a3bf0c675ffdf00c6d2fabe1a5df1115e80adce
+```
+
+The `SIGNED TAG` is the signed image tag with a unique content-addressable `DIGEST`. `SIGNERS` lists all entities who have signed.
+
+The administrative keys listed specify the root key of trust, as well as the administrative repository key. These keys are responsible for modifying signers, and rotating keys for the signed repository.
```
Comment (thaJeztah) [inline docs/reference/commandline/trust_inspect.md]: can you wrap these lines to 80 chars?

File: docs/reference/commandline/trust_inspect.md [hunk 4]
```diff
@@ -362,3 +357,102 @@ $ docker trust inspect alpine notary
...
+
+SIGNER              KEYS
+alice               47caae5b3e61, a85aab9d20a4
+bob                 034370bcbd77, 82a66673242c
+carol               b6f9f8e1aab0
+
+Administrative keys for my-image:
+Repository Key:	27df2c8187e7543345c2e0bf3a1262e0bc63a72754e9a7395eac3f747ec23a44
+Root Key:	40b66ccc8b176be8c7d365a17f3e046d1c3494e053dd57cfeacfe2e19c4f8e8f
+```
```
Comment (thaJeztah) [inline docs/reference/commandline/trust_inspect.md]: please add a newline at the end

**Comment by vieux** [comment]
@thaJeztah I addressed your comments

**Pull request merged**